go/vt/mysqlctl/s3backupstorage/s3.go

```diff
@@ -224,7 +241,17 @@ func calculateUploadPartSize(filesize int64) int64 {
 			partSizeBytes = calculatedPartSizeBytes
 		}
 	}
-	return partSizeBytes
+
+	if minPartSize != 0 && partSizeBytes < minPartSize {
+		if minPartSize > 1024*1024*1024*5 || minPartSize < 1024*1024*5 { // 5GiB and 5MiB respectively
```

Comment:
```suggestion
		if minPartSize > MaxPartSize || minPartSize < 1024*1024*5 { // 5GiB and 5MiB respectively
```

Reply:
that was always the intention after the earlier comments in this PR, I think I left it into an uncommited change that I wiped out during the rebase. We should also use `manager.MinUploadPartSize` here